src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -308,8 +327,8 @@ def send_and_receive(
 
     def close(self) -> None:
         """Disconnect from the SuperLink if connected."""
-        # Check if GrpcDriverHelper is initialized
-        if self.driver_helper is None:
+        # Check if is initialized
+        if self._run is None:
```

Comment:
Shouldn't this be removed? In `__init__`, we ensure that `self.stub` is never `None`, so we don't need to check for it